code/menuui/credits.cpp

```diff
@@ -260,6 +260,20 @@ void credits_start_music()
 	}
 }
 
+char* credits_get_music_filename(const char* music)
```

Comment:
this should really return a ``const char*``, since there is really no usecase in which you want to be able to modify the name of the music filename.